docker/runtests.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
+DOCKER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$DOCKER_DIR/shared.env"
+
 set -e -x
 
-for V in cp36-cp36m cp35-cp35m cp27-cp27m cp27-cp27mu; do
+for V in "${PYTHON_VERSIONS[@]}"; do
```

Comment:
Is there any reason this isn't:

```suggestion
for PYBIN in /opt/python/*/bin; do
```